Silicon/AlderlakePkg/Library/HeciMeExtLib/HeciMeExtLib.c

```diff
@@ -403,3 +403,204 @@ HeciRevokeOemKey (
 
   return Status;
 }
+
+//
+// MKHI_MCA_GROUP_ID = 0x0A
+//
+
+/**
+  This message is sent by the BIOS to retrieve file stored in ME firmware NVM.
+
+  @param[in]      FileId          Id number of file to read
+  @param[in]      Offset          File offset
+  @param[in, out] DataSize        On input - size of data to read, on output - size of read data
+  @param[in]      Flags           Flags
+  @param[out]     *Buffer         Pointer to the data buffer
+
+  @retval EFI_UNSUPPORTED         Current ME mode doesn't support this function
+  @retval EFI_SUCCESS             Command succeeded
+  @retval EFI_DEVICE_ERROR        HECI Device error, command aborts abnormally
+  @retval EFI_TIMEOUT             HECI does not return the buffer before timeout
+  @retval EFI_ABORTED             Cannot allocate memory
+  @retval EFI_BUFFER_TOO_SMALL    Message Buffer is too small for the Acknowledge
+**/
+EFI_STATUS
+EFIAPI
+HeciReadFileExMsg (
+  IN UINT32      FileId,
+  IN UINT32      Offset,
+  IN OUT UINT32  *DataSize,
+  IN UINT8       Flags,
+  OUT UINT8      *DataBuffer
```

Comment:
better update the name in the comment to match with this name.

Reply:
Done